src/modules/NdviColormap/Module.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Sample Meta Module for demonstration purpose only
+ */
+module.exports = function NdviColormapfunction() {
+    this.expandSteps([{ 'name': 'ndvi', 'options': {} }, { 'name': 'colormap', options: {} }]);
```

Comment:
I guess I'm wondering one last thing, could we create a metal module from a json object with createMetaModule right here, alternatively? Any pros or cons to that rather than using expandSteps? Cool!